extensions/opentelemetry/deployment/pom.xml

```diff
@@ -122,21 +122,21 @@
             <scope>test</scope>
         </dependency>
 
-        <dependency>
-            <groupId>io.opentelemetry.contrib</groupId>
-            <artifactId>opentelemetry-aws-xray-propagator</artifactId>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>io.opentelemetry.contrib</groupId>
-            <artifactId>opentelemetry-aws-resources</artifactId>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>io.opentelemetry.contrib</groupId>
-            <artifactId>opentelemetry-aws-xray</artifactId>
-            <scope>test</scope>
-        </dependency>
+<!--        <dependency>-->
```

Comment:
Either you remove them entirely or if you let them commented out, please add a comment explaining why they are commented and still around. Future us will thank you :).